### ISSUE TITLE
`--no-ignore` does not include inline ignored violations

### ISSUE THREAD

**Issue by juliahw** [get-woke/woke#54]
`--no-ignore` should report inline ignored violations in addition to violations in ignored files.

- [x] Yes, I've searched similar issues on GitHub and didn't find any.
- [x] Yes, I've included all information below (version, config, etc).

Please include the following information:

<details><summary>Version of woke</summary>

```console
$ woke --version
0.5.0
```

</details>

<details><summary>Config file</summary>

N/A; this bug is reproducible using default rules.

</details>

<details><summary>Go environment</summary>

N/A; this bug is reproducible in any environment.

</details>

<details><summary>Verbose output of running</summary>

```console
$ cat foo
Hey, check out this whitelist! // wokeignore:rule=whitelist
$ woke --debug --no-ignore
2021-03-04T19:24:22-08:00 DBG woke version 0.5.0 built from 8917e8e on 2021-02-10T02:07:06Z
2021-03-04T19:24:22-08:00 DBG rules enabled rules=["blackbox","blacklist","dummy","grandfathered","guys","man-hours","master-slave","sanity","slave","whitebox","whitelist"]
2021-03-04T19:24:22-08:00 DBG created new printer printer=text
2021-03-04T19:24:22-08:00 DBG process files path=. type=parallel
2021-03-04T19:24:22-08:00 DBG skipping file=. reason="file is a directory"
2021-03-04T19:24:22-08:00 DBG ignoring via in-line file=foo line=1 rule=whitelist
2021-03-04T19:24:22-08:00 DBG finished processing violations durationMS=0.38887 file=foo
No violations found. Stay woke ✊
2021-03-04T19:24:22-08:00 DBG woke completed durationMS=1.493173
$ woke --debug
2021-03-04T19:24:29-08:00 DBG woke version 0.5.0 built from 8917e8e on 2021-02-10T02:07:06Z
2021-03-04T19:24:29-08:00 DBG rules enabled rules=["blackbox","blacklist","dummy","grandfathered","guys","man-hours","master-slave","sanity","slave","whitebox","whitelist"]
2021-03-04T19:24:29-08:00 DBG skipping ignorefile error="open .gitignore: no such file or directory" file=.gitignore
2021-03-04T19:24:29-08:00 DBG skipping ignorefile error="open .wokeignore: no such file or directory" file=.wokeignore
2021-03-04T19:24:29-08:00 DBG finished compiling ignores durationMS=0.080036
2021-03-04T19:24:29-08:00 DBG created new printer printer=text
2021-03-04T19:24:29-08:00 DBG process files path=. type=parallel
2021-03-04T19:24:29-08:00 DBG skipping file=. reason="file is a directory"
2021-03-04T19:24:29-08:00 DBG ignoring via in-line file=foo line=1 rule=whitelist
2021-03-04T19:24:29-08:00 DBG finished processing violations durationMS=0.174957 file=foo
No violations found. Stay woke ✊
2021-03-04T19:24:29-08:00 DBG woke completed durationMS=1.112633
```

</details>

**Comment by caitlinelfring** [comment]
The description of `--no-ignore` flag is `Files matching entries in .gitignore/.wokeignore are parsed`, so it is doing as it's intended, to process _files_ through woke that are intended to be ignored.

That said, I can see how skipping processing of inline ignored lines while files aren't skipped with `--no-ignore` feels like unexpected behavior

**Comment by caitlinelfring** [comment]
@juliahw The fix for this has been released in `v0.6.0` https://github.com/get-woke/woke/releases/tag/v0.6.0. Thanks for reporting!
